Source/controls/plrctrls.cpp

```diff
@@ -102,24 +100,23 @@ int GetDistance(int dx, int dy, int maxDistance)
 
 /**
  * @brief Get distance to coordinate
- * @param dx Tile coordinates
- * @param dy Tile coordinates
+ * @param destination Tile coordinates
  */
-int GetDistanceRanged(int dx, int dy)
+int GetDistanceRanged(Point destination)
 {
-	int a = plr[myplr].position.future.x - dx;
-	int b = plr[myplr].position.future.y - dy;
-
-	return sqrt(a * a + b * b);
+	return plr[myplr].position.future.ExactDistance(destination);
 }
 
 void FindItemOrObject()
 {
+	// The loops in this function are iterating through tiles offset from the player position. This
+	// could be accomplished by looping over the values in the Direction enum and making use of
+	// Point math instead of nested loops from [-1, 1].

```

Comment:
Doxygen:
```suggestion
/**
 * @todo The loops in this function are iterating through tiles offset from the player position. This
 * could be accomplished by looping over the values in the Direction enum and making use of
 * Point math instead of nested loops from [-1, 1].
 */
void FindItemOrObject()
{
```